.github/workflows/sorn-grab.yml

```diff
@@ -0,0 +1,20 @@
+name: "Find SORNs"
```

Comment:
is there are reason to call this `sorn grab` whne the name of it is `Find SORNs`? seems like having naming consistency might be a bit easier to follow.

Reply:
I put a comment up at the top of the pr
> I wanted to change the sorn-grab.yml filename, but there [may be a bug](https://github.community/t/workflow-dispatch-workflow-not-showing-in-actions-tab/130088) that will prevent it from running if we change it. Specifically will prevent the "Run workflow" button from appearing. Maybe, not entirely sure.

The bug is that github actions will register a branches workflow with a filename. If it get changed then it won't run. I did change the file and it wouldn't run anymore, until I changed the filename back.

We can change the file name after if it works.

Reply:
ok, makes sense. I can't. approve the pr because i originally opened it. But I think we're good to merge